docker-for-mac/install.md

```diff
@@ -1,96 +1,88 @@
+virtualization system running (HyperKit) which takes the place of the
+VirtualBox system. To learn more, see [Docker Desktop for Mac vs. Docker Toolbox](docker-toolbox.md).
 
-* **System Requirements**: Docker Desktop for Mac launches only if all of these
-  requirements are met.
+## System requirements
 
-  - Mac hardware must be a 2010 or newer model, with Intel's hardware support for memory
-    management unit (MMU) virtualization, including Extended Page Tables (EPT) and
-    Unrestricted Mode. You can check to see if your machine has this support by
-    running the following command  in a terminal: `sysctl kern.hv_support`
+Your Mac must meet the following requirements to successfully install Docker Desktop:
 
-  - macOS Sierra 10.12 and newer macOS releases are supported. We recommend
-    upgrading to the latest version of macOS.
+- Mac hardware must be a 2010 or newer model, with Intel’s hardware support for memory management unit (MMU) virtualization, including Extended Page Tables (EPT) and Unrestricted Mode. You can check to see if your machine has this support by running the following command in a terminal: `sysctl kern.hv_support`
 
-  - At least 4GB of RAM
+- macOS must be version 10.12 or newer. We recommend upgrading to the latest version of macOS.
 
-  - VirtualBox prior to version 4.3.30 must NOT be installed (it is incompatible
-    with Docker Desktop for Mac). If you have a newer version of VirtualBox installed, it's fine.
+  **Note:** Docker supports Docker Desktop on the most recent versions of macOS. That is, the current release of macOS and the previous two releases. As new major versions of macOS are made generally available, Docker will stop supporting the oldest version and support the newest version of macOS (in addition to the previous two releases).
 
-  > **Note**: If your system does not satisfy these requirements, you can
-  > install [Docker Toolbox](/toolbox/overview.md), which uses Oracle VirtualBox
-  > instead of HyperKit.
+- At least 4GB of RAM.
 
-* **What the install includes**: The installation provides
+- VirtualBox prior to version 4.3.30 must not be installed as it is not compatible with Docker Desktop.
+
+**Note**: If your system does not satisfy these requirements, you can install [Docker Toolbox](/toolbox/overview.md), which uses Oracle VirtualBox instead of HyperKit.
+
+## What's included in the installer
+
+The Docker Desktop installation includes
   [Docker Engine](/engine/userguide/), Docker CLI client,
   [Docker Compose](/compose/), [Docker Machine](/machine/overview/), and [Kitematic](/kitematic/userguide.md).
 
-## Install and run Docker Desktop for Mac
+## Install and run Docker Desktop on Mac
 
-1.  Double-click `Docker.dmg` to open the installer, then drag Moby the whale to
+1. Double-click `Docker.dmg` to open the installer, then drag Moby the whale to
```

Comment:
Suggest changing "Moby the whale" to "the icon"

Reply:
Updated to '..drag the Docker icon to...'